controller/projects.js

```diff
@@ -47,35 +46,13 @@ exports.createProject = async (ctx, next) => {
   const proj = ctx.request.body;
   console.log('REQ.BODY', proj);
   try {
-    await project
-      .create(
-        {
-          name: proj.name,
-          category: proj.category,
-          organizationName: proj.organizationName,
-          organizationId: proj.organizationId,
-          address: proj.address,
-          city: proj.city,
-          date: proj.date,
-          shortTerm: proj.shortTerm,
-          time: proj.time,
-          description: proj.description,
-          role: proj.role,
-          qualifications: proj.qualifications,
-          tasks: proj.tasks,
-          totalSpaces: proj.totalSpaces,
-        },
-        {
-          returning: true,
-        },
-      )
-      .then(project => {
-        console.log(
-          project.get({
-            plain: true,
-          }),
-        );
-      });
+    await Project.create({ ...proj }, { returning: true }).then(project => {
```

Comment:
You can remove the returning flag here, as it is set to "true" by default :). 
Also if I see correctly due to the console.log statement this function does not return project back to the client. Let's check :).